fw/http_match.c

```diff
@@ -963,6 +963,9 @@ tfw_http_search_host_forwarded(const TfwStr *hdr, TfwStr *host_val)
 {
 	TfwStr *chunk, *end;
```

Comment:
I second that. But don't forget to remove related validation from `frang_http_host_check()` under `TFW_HTTP_VER_11` switch label.

Reply:
Yeah, I agree to remove the logic. I revised the original #1350 and it doesn't require `Forward: host=...` to match authority. It seem this was done by the comment https://github.com/tempesta-tech/tempesta/pull/1629/commits/26e1aeba601cd45404ab66ac028511cc33b4b0dc#r877562638 and it looks wrong. I also don't see a case when some original Host can be rewritten to the same value (the use case "This can be used, for example, by the origin server if a reverse proxy is rewriting the "Host" header field to some internal host name." from https://datatracker.ietf.org/doc/html/rfc7239#section-5.3 also means different authorities).

Reply:
FIxed